pkg/replication/replication.go

```diff
@@ -378,6 +407,8 @@ func (r *replication) ensureHealthy(
 		select {
 		case <-quitCh:
 			return
+		case <-r.replicationCancelledCh:
```

Comment:
Actually this `quitCh` is passed by a parameter, it's actually a quitCh for the timeout, it's not related to `r.quitCh`